### PR TITLE
Test cases

### PR DESCRIPTION
**Pull request by samuelbarrett**
* fix vehicle capacity to a static amount, and cap location demand to that amount so each location can be served by a single vehicle
* added tests for different fleet sizes as a fraction of the location count. Runs for all algorithms.
* fixed `two_opt` algorithm to allow use of only some vehicles if applicable
* added graph creation of problem set without the routes solutions drawn
* saves graph outputs from tests to PNG in `src/data/figures/title.png`

### CODE REVIEW

File: src/benchmark.py [hunk 1]
```diff
@@ -179,77 +179,74 @@ def test_small_map():
...
+  return model
 
 
 def benchmark_suite():
   """
   Run a series of tests on each algorithm and save the results
   """
   print('starting benchmarks...')
-  test_small_map()
+  # this is the temporary code for debugging why or_tools is returning a None solution... hmm
```
Comment (VictoriaKGN) [inline src/benchmark.py]: might want to remove this

File: src/benchmark.py [hunk 2]
```diff
@@ -179,77 +179,74 @@ def test_small_map():
...
+  # model, coords = create_model(1000, 10, 5, [200]*5)
+  # routes, distance, exec_time = test_guided_local_search(model)
+  
+  # This is the real test, different fleets and locations
+  fleet_ratios = [0.75, 0.5, 0.25, 0.15]
+  location_counts = [100]
```
Comment (VictoriaKGN) [inline src/benchmark.py]: also here, should we put all the counts: 10, 50, 100, 200 here?

File: src/benchmark.py (hunk 1 above)
Comment (samuelbarrett) [reply src/benchmark.py]: removed thanks!

File: src/benchmark.py (hunk 2 above)
Comment (samuelbarrett) [reply src/benchmark.py]: done

**Review by VictoriaKGN — APPROVED** [review]

**Pull request merged**